apps/docs-website/docusaurus.config.ts

```diff
@@ -22,7 +22,7 @@ const config: Config = {
   baseUrl: '/',
 
   future: {
-    experimental_faster: true,
+    experimental_faster: false,
```

Comment:
FYI, I fixed this over in #305, @AndreaPontrandolfo.

Reply:
I tried it earlier, but it didn't work for me.

Reply:
ye i tried the override, but it didnt work.

Reply:
Can you try now, after refreshing the lockfile? rspack 1.1.1 should've fixed it.